apps/pie-docs/src/engineers/guidelines/browser-support.md

```diff
@@ -12,41 +12,41 @@ eleventyComputed:
 
 ## Browser support
 
-The number of browsers available to our users is vast, especially for those using mobile devices. Below, we have defined a rating system that helps us to specify the level of browser support we aim to achieve and to clarify what level of testing is required.
+Users have access to hundreds of browsers, so instead of assessing them all individually we've come up with a rating system which defines different levels of browser support and the testing requirements for each level. We've assigned the highest ratings to the browsers that are most commonly used by our customers.
```

Comment:
Without trying to be pedantic, `Users have access to hundreds of browsers` isn't really true. They have access to hundreds of versions of browsers, but not actual browser variations. Just doesn't read correctly to me having gone through the copyrighting.

Reply:
I've updated it to 'hundreds of versions of browsers'